src/device/OS.js

```diff
@@ -134,7 +134,7 @@ function init ()
         OS.cordova = true;
     }
     
-    if ((typeof process !== 'undefined') && (typeof process.versions.node !== 'undefined'))
+    if ((typeof process !== 'undefined') && (typeof process.versions !== 'undefined') && (typeof process.versions.node !== 'undefined'))
```

Comment:
We could probably shorten this to: `if (process && process.versions && process.versions.node)`